lib/xcsoar/mapgen/waypoints/seeyou_reader.py

```diff
@@ -75,27 +77,65 @@ def __parse_length(str):
 
 def parse_seeyou_waypoints(lines, bounds=None):
     waypoint_list = WaypointList()
+    cherrypy.log('in parse_seeyou_waypoints function:')
 
-    first = True
-    for line in lines:
-        if first:
-            first = False
-            continue
+    #gfp 241210: modified to wait for header line before processing
+    #gfp 241210: added 'ISO-8859-2' decoding for correct cherrypy logging display
+
+    #gfp 241208 added to print out all lines in selected .CUP file
+    # wpnum = 0
+    # for byteline in lines:
+    #     wpnum = wpnum + 1
+    #     line = byteline.decode('ISO-8859-2')
+
+    header = 'name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc'
 
+    wpnum = 0
+    for byteline in lines:
+        wpnum = wpnum + 1
+        line = byteline.decode('ISO-8859-2') #gfp 241210: added 'ISO-8859-2' decoding for correct cherrypy logging display
```

Comment:
Shouldn't this be UTF-8?

Reply:
I tried 'UTF-8' and AFAICT, it doesn't handle the accent marks in the Slovenia3 waypoint list.  I added the 'Slovenia3.cup' and 'Slovenia3.dat' test files for your consideration (the .DAT file was generated from the .CUP file using a separate Python script to make sure they both contained the exact same coordinates)

I have no problem using UTF-8 vs ISO-8859-2, but some of the characters in Slovenia3.cup won't be displayed properly. I don't think this will affect the bounds calculation either way.

Reply:
The files you provide contain corrupted data.
Slovenia3.cup: Lines 26, 94, 774, 776. 

A CUP file should be utf-8. https://github.com/naviter/seeyou_file_formats/blob/main/CUP_file_format.md

These files should only be included if they serve an automated test case, and then they should be under test/data/cup/filename.

Reply:
https://download.xcsoar.org/content/waypoint/country/SI-WPT-National-OpenAIP.cup if you need an example that works.

Reply:
https://www.condorsoaring.com/forums/viewtopic.php?f=29&t=22737&p=191905#p191905 seems to be the file from slovenia3 in condor.